modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -308,17 +154,19 @@ Grant [command]``sudo`` access to the user by adding this line to the [filename]
 ====
 This procedure grants root access without requiring a password, which is required for registering the client.
 When the client is successfully installed it will run with root privileges, so the access is no longer required.
-We recommend that you remove the line from the `sudoers` file after the client has been successfully installed.
+We recommend that you remove the line from the [path]``sudoers`` file after the client has been successfully installed.
 ====
 
+
+
 == Register Clients
 
 To register your {ubuntu} clients, you need a bootstrap repository.
 By default, bootstrap repositories are automatically created, and regenerated daily for all synchronized products.
 You can manually create the bootstrap repository from the command prompt, using this command:
 
 ----
-mgr-create-bootstrap-repo --with-custom-channels
+mgr-create-bootstrap-repo
```

Comment:
As I said, I am not 100% sure about this. Check with Michael before marging.

Reply:
@mcalmer can you please advise?

Reply:
for 16.04 and 18.04 you need it. For 20.04 it is not needed